Created Readme file (#189)
o3de/community: process-docs-and-markdown/README.md

# Markdown & Process Guides

> The following is a compilation of guides, procedural documents, and illustrative examples intended for the community to support the creation of new content and updating existing content with O3DF's best practices. These resources are dynamic, regularly updated to reflect changes in processes. If there are any errors or omissions, please create an issue to report them, so they can be corrected.

Reviewer: ```suggestion
This directory contains a compilation of guides, procedural documents, and illustrative examples made for the community to support the creation of new content and maintenance of existing content using O3DF's best practices. These resources are dynamic and regularly updated to reflect changes in processes. If there are any errors or omissions, please create an issue to report them, so they can be corrected.
```

> ## GitHub markdown

Reviewer: ```suggestion
## GitHub Markdown
```

> This manual is produced to assist members who are interested in establishing GitHub repositories or maintaining current ones and seeking to explore their true capabilities, along with step-by-step instructions and illustrative examples.

Reviewer: ```suggestion
This manual was created to assist members who are interested in establishing GitHub repositories or maintaining current ones and seeking to explore their true capabilities, with step-by-step instructions and illustrative examples.
```

[GitHub Markdown](/process-docs-and-markdown/github-markdown.md)

## Blog Process Guides

> These resources are available to offer support and direction on optimal approaches for constructing and sustaining blogs for the O3DE Website. This document will be regularly revised to reflect any modifications made over time.

Reviewer: ```suggestion
These resources are available to offer support and direction on optimal approaches for constructing and maintaining blogs for the O3DE Website. This document will be regularly revised to reflect any modifications made over time.
```

[Blog Process Guide](/process-docs-and-markdown/blog-process.md)<br>
[O3DE Blog Cheatsheet](/process-docs-and-markdown/website-markdown.md)<br>
[Templates and Examples](/process-docs-and-markdown/files/)

## O3DE Website Maintenance

> These resources offer support in generating code, banners, and color schemes for the website. Additionally, they provide instructions on updating images, adding new foundation members, and revising different sections of the o3de website.

Reviewer: ```suggestion
These resources assist with generating code, banners, and color schemes for the website. Additionally, they provide instructions on updating images, adding new foundation member logos, and revising different sections of the O3DE website.
```
